test/gtest/uct/test_flush.cc

```diff
@@ -163,7 +161,7 @@ class uct_flush_test : public uct_test {
         mapped_buffer recvbuf(length, SEED2, receiver());
         sendbuf.pattern_fill(SEED3);
         blocking_put_bcopy(sendbuf, recvbuf);
-        (this->*flush)();
+        (this->*flush)(m_flush_flags);
```

Comment:
can we remove the member variable m_flush_flags?

Reply:
> can we remove the member variable m_flush_flags?

no: it is used in completion callback and we have to store it till end-of-op inside test context

Reply:
i see, all existing tests already set m_flush_flags in the test :(
so better revert it, and remove the flags parameter to flush_func_t

Reply:
> so better revert it, and remove the flags parameter to flush_func_t

ok, reverted